solana/programs/token-bridge-relayer/src/processor/inbound.rs

```diff
@@ -160,6 +155,22 @@ pub fn complete_transfer(mut ctx: Context<CompleteTransfer>) -> Result<()> {
         TokenBridgeRelayerError::InvalidRecipient
     );
 
+    // Initialize the temporary account:
+    CreateAndInitTokenAccount {
+        system_program: ctx.accounts.system_program.to_account_info(),
+        token_program: ctx.accounts.token_program.to_account_info(),
+        rent: ctx.accounts.rent.clone(),
+        payer: ctx.accounts.payer.to_account_info(),
+        account: ctx.accounts.temporary_account.to_account_info(),
+        mint: ctx.accounts.mint.to_account_info(),
+        authority: ctx.accounts.wormhole_redeemer.to_account_info(),
+    }
+    .run_with_seeds(&[
+        SEED_PREFIX_TEMPORARY,
+        ctx.accounts.mint.to_account_info().key.as_ref(),
+        &[temporary_account_bump],
+    ])?;
```

Comment:
What's the difference between this and just using the `init` in the context? You could pass the `temporary_account_bump` in there as instruction data too.

Reply:
I put a comment explaining this: Anchor forbids doing so, for "security reasons".